### PR TITLE
Website Updates for 0.2.0

### PR DESCRIPTION
**Pull request by DarinJ**

### CODE REVIEW

File: website/_posts/blogs/2016-06-29-myriad-0.2.0-release-announcement.md [hunk 1]
```diff
@@ -0,0 +1,57 @@
...
+title: Apache Myriad 0.2.0 (Incubating) Released
+author: <a href="https://github.com/darinj">Darin Johnson</a> (Apache Myriad committer, 0.2.0 Release Manager)
+date:   2016-06-29 00:00:00
+categories: blogs
+---
+
+##### {{page.title}} 
+
+On behalf of the entire Myriad Team, I’m happy to announce the release of Apache Myriad 0.2.0 (incubating).  This is the 
+second release of Apache Myriad.  Among the features of the 0.2.0 release are increased reliability of Fine Grained Scaling, support for running the Node
```
Comment (adam-mesos) [inline website/_posts/blogs/2016-06-29-myriad-0.2.0-release-announcement.md]: "second release of Myriad in the Apache Incubator."

File: website/_posts/blogs/2016-06-29-myriad-0.2.0-release-announcement.md [hunk 2]
```diff
@@ -0,0 +1,57 @@
...
+author: <a href="https://github.com/darinj">Darin Johnson</a> (Apache Myriad committer, 0.2.0 Release Manager)
+date:   2016-06-29 00:00:00
+categories: blogs
+---
+
+##### {{page.title}} 
+
+On behalf of the entire Myriad Team, I’m happy to announce the release of Apache Myriad 0.2.0 (incubating).  This is the 
+second release of Apache Myriad.  Among the features of the 0.2.0 release are increased reliability of Fine Grained Scaling, support for running the Node 
+Manager and Auxiliary Services (Jobhistory Server, etc) in Docker Containers, and better distribution management of the
```
Comment (adam-mesos) [inline website/_posts/blogs/2016-06-29-myriad-0.2.0-release-announcement.md]: "JobHistory"?

File: website/_posts/blogs/2016-06-29-myriad-0.2.0-release-announcement.md [hunk 3]
```diff
@@ -0,0 +1,57 @@
...
+* Sarjeet Singh
+* Shingo Omura
+* Suneel Marthi
+* Swapnil Daingade
+* Yuliya Feldman
+* Zachary Jaffee
+* Zhongyue Luo
+ 
+This is the second of many Apache Myriad releases. We've already started merging new pull requests for the next release, 
+where we plan to increase tackle YARN Security and adding additional Mesos features into Myriad such as the ability to use
```
Comment (adam-mesos) [inline website/_posts/blogs/2016-06-29-myriad-0.2.0-release-announcement.md]: "we plan to add support for YARN Security and advanced Mesos features like reservations/volumes and oversubscription."

File: website/_posts/news/2016-06-29-myriad-0.2.0-release-announcement.md [hunk 4]
```diff
@@ -0,0 +1,57 @@
...
+layout: big_card_for_md
+title: Apache Myriad 0.2.0 (Incubating) Released
+author: <a href="https://github.com/darinj">Darin Johnson</a> (Apache Myriad committer, 0.2.0 Release Manager)
+date:   2016-06-29 00:00:00
+categories: blogs
+---
+
+##### {{page.title}} 
+
+On behalf of the entire Myriad Team, I’m happy to announce the release of Apache Myriad 0.2.0 (incubating).  This is the
```
Comment (adam-mesos) [inline website/_posts/news/2016-06-29-myriad-0.2.0-release-announcement.md]: Seems weird that you have to commit this in two places, but whatevs.

**Comment by adam-mesos** [comment]
Minor rewordings, but otherwise :+1: :rocket:

File: website/_posts/news/2016-06-29-myriad-0.2.0-release-announcement.md (hunk 4 above)
Comment (DarinJ) [inline website/_posts/news/2016-06-29-myriad-0.2.0-release-announcement.md]: Yeah wasn't pleased but didn't want to fight it.  This is what Santosh did as well.

**Comment by DarinJ** [comment]
Great will wait for others to weigh in, will probably just take the suggestions and then update.

File: website/_posts/news/2016-06-29-myriad-0.2.0-release-announcement.md (hunk 4 above)
Comment (smarella) [inline website/_posts/news/2016-06-29-myriad-0.2.0-release-announcement.md]: @DarinJ, there are two files with the same name, but different content.

This is the actual blog post: 
https://github.com/apache/incubator-myriad/blob/master/website/_posts/blogs/2015-12-09-myriad-0.1.0-release-announcement.md

This is the news feed that has links to the blog post and download page: 
https://github.com/apache/incubator-myriad/blob/master/website/_posts/news/2015-12-09-myriad-0.1.0-incubating-released.md

**Comment by smarella** [comment]
The content looks good to me. 👍  

Unless already done, please run "jekyll s" locally and browse the website to make sure the changes appear as you'd like them to.

File: website/_posts/news/2016-06-29-myriad-0.2.0-release-announcement.md (hunk 4 above)
Comment (DarinJ) [inline website/_posts/news/2016-06-29-myriad-0.2.0-release-announcement.md]: Ah, now I see.  Will you a similar new/blog format.

**Comment by smarella** [comment]
ship it 🚀

**Comment by sdaingade** [comment]
LGTM. Ship it!

**Comment by DarinJ** [comment]
Pushed to svn so the website is live, will git merge later as I don't have time right now.

**Comment by DarinJ** [comment]
Merged so closing!

**Pull request closed without merging**